packages/untp-test-suite/src/templates/utils/path.ts

```diff
@@ -0,0 +1,10 @@
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+export const getCurrentFilePath = () => {
+  return fileURLToPath(import.meta.url);
+};
+
+export const getCurrentDirPath = (filePath: string) => {
+  return dirname(filePath);
```

Comment:
The code is created in the previous PR, should we utilize it and reuse it for the previous code?

Reply:
Yes, I agree. I've just updated the credentials utility functions to utilize this code, and then updated their unit tests as well.